portfolio/src/main/webapp/script.js

```diff
@@ -81,8 +81,39 @@ function hasWon(){
     }
 }
 
-async function getHello() {
-  const response = await fetch('/data');
-  const hello = await response.text();
-  document.getElementById('hello-container').innerText = hello;
-}
\ No newline at end of file
+/**
+ * Fetches the current state of the game and builds the UI.
+ */
+async function getCommentSection() {
+  // Clear history of comments. 
+  const history = document.getElementById('history');
+  while(history.firstChild) {
+    history.removeChild(history.firstChild);
+  }
+
+  // Get user's desired number of comments. 
+  var selectedMaxComments = document.getElementById('exampleFormControlSelect1').value;
+  var url = '/data?maxComments=' + selectedMaxComments;
```

Comment:
Can these be `const`s? In general, it's good practice to use `const` whenever possible.

Reply:
Yes, fixed. Thanks, I didn't know this